tests/snappi_tests/multidut/pfc/files/lossless_response_to_external_pause_storms_helper.py

```diff
@@ -316,7 +317,12 @@ def __gen_data_flow(testbed_config,
         flow = testbed_config.flows.flow(name='{} {} -> {}'.format(flow_name_prefix, src_port_id, dst_port_id))[-1]
         flow.tx_rx.port.tx_name = testbed_config.ports[src_port_id].name
         flow.tx_rx.port.rx_name = testbed_config.ports[dst_port_id].name
-        eth, ipv4 = flow.packet.ethernet().ipv4()
+        eth, ipv4, udp = flow.packet.ethernet().ipv4().udp()
+        src_port = random.randint(5000, 6000)
+        udp.src_port.increment.start = src_port
+        udp.src_port.increment.step = 1
+        udp.src_port.increment.count = 1
```

Comment:
if the total count is 1, there is still 50% chance that two flows may be sent on the same physical link. maybe try 100?

Reply:
we can leave this as it is for now. and monitor it for further improvements when needed.

Reply:
If traffic on each queue <= 200G, we can remove random and set fixed udp port for each queue.
Something like src_port = 5000 + queue_id